extensions/amp-autocomplete/validator-amp-autocomplete.protoascii

```diff
@@ -0,0 +1,102 @@
+#
```

Comment:
We should not merge the validation rules yet. Eventhough this is under experimental and does not work, if validation rules are merged, people can create (maybe unknowingly*) valid AMP pages that has non-working `amp-autocomplete` in it and if we change the API, we will break their whole page.

It is just safer to wait until the end to merge validation rules.

*injected by some CMS, they were trying it and forgot to remove it, etc..